portfolio/src/main/webapp/index.html

```diff
@@ -81,7 +81,22 @@ <h1>Adam's Personal Portfolio</h1>
           <li>Friends and family also make an appearance!</li>
         </ul>  
       </p>
-      <button onclick="getData()">Get the Data URL</button>
+
+      <div id="comment">
+        <h2>Leave a Comment</h2>
+        <form action="/data" method="POST">
```

Comment:
nit: these changes which add a form and connect it to the POST endpoint should probably be in the same CL which defines the doPost method on the server, since sending that message from the client and processing it on the server are part of the same logical change.

Reply:
Yesterday I accidentally clicked the wrong walk through so I added those lines. They stay the same in Server-POST. I'd be happy to remove them and push a new version but that may cause merge conflicts Server-POST (the CL for the next step). Should I do remove them anyway?

Reply:
No need to remove them! Mostly I wanted to bring up the point about keeping PR's conceptually related, but sounds like you're already on board with that. No action needed.